pom.xml

```diff
@@ -347,8 +347,8 @@
       <properties>
         <sparkbundle.version>3.5</sparkbundle.version>
         <sparkshim.artifactId>spark-sql-columnar-shims-spark35</sparkshim.artifactId>
-        <spark.version>3.5.2</spark.version>
-        <iceberg.version>1.5.0</iceberg.version>
+        <spark.version>3.5.5</spark.version>
+        <iceberg.version>1.8.0</iceberg.version>
```

Comment:
should we also enable jdk17 here?

Reply:
We can’t define JDK 17 here. If we define the `java.version` to 17 in the profile `spark-3.5`, then `mvn -Pspark-3.5 -Pjava-8` and `mvn -Pjava-8 -Pspark-3.5` would exhibit different behaviors.